scvi/external/sysvi/_base_components.py

```diff
@@ -0,0 +1,345 @@
+from __future__ import annotations
+
+from collections import OrderedDict
+from typing import Literal
+
+import torch
+from torch.distributions import Normal
+from torch.nn import (
+    BatchNorm1d,
+    Dropout,
+    LayerNorm,
+    Linear,
+    Module,
+    Parameter,
+    ReLU,
+    Sequential,
+)
+
+
+class Embedding(Module):
```

Comment:
Can you switch this to use the Embedding class within scvi-tools. Should perform similarly and otherwise we can add options there.